src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -367,8 +370,27 @@ public ResultWrapper<BlockForRpc> eth_getBlockByNumber(BlockParameter blockParam
         return GetBlock(blockParameter, returnFullTransactionObjects);
     }
 
+    private bool IsBlockPruned(BlockParameter blockParameter)
+    {
+        if (_syncConfig.AncientReceiptsBarrierCalc > 1 && _syncConfig.AncientBodiesBarrierCalc > 1)
```

Comment:
Is it `1` or `0`?

Reply:
For some reason, it's 1 🤷  https://github.com/NethermindEth/nethermind/blob/3254dc2a85afbde59e64b292a51b82bd9e75522b/src/Nethermind/Nethermind.Blockchain/Synchronization/ISyncConfig.cs#L90